dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -16,8 +16,50 @@
 
 # ACME configuration
 
-#configParm1=Configuration parameter 1 for ACME
-#configParm1.desc=Description for parameter 1
+directoryURI=ACME server directory URI
+directoryURI.desc=The URI to the ACME CA server's directory object.
 
-#configParm2=Configuration parameter 2 for ACME
-#configParm2.desc=Description for parameter 2
+domain=Domain name
+domain.desc=A domain name to request a certificate for.
+
+validFor=Valid for
+validFor.desc=The duration of time that the certificate signing request specifies for the certificate to be valid. The default is defined by the ACME CA server.
+
+country=Country
+country.desc=Country name to use for the certificate.
+
+state=State or province
+state.desc=State or province name to use for the certificate.
+
+locality=Locality
+locality.desc=Locality name to use for the certificate.
+
+organization=Organization
+organization.desc=Organization name to use for the certificate.
+
+organizationalUnit=Organizational unit
+organizationalUnit.desc=Organizational unit name to use for the certificate.
+
+acceptTermsOfService=Accept terms of service
+acceptTermsOfService.desc=Indicates whether the user accepts the terms of service for the ACME CA server. If true, the terms of service are automatically accepted. If false, and the ACME CA server issues terms of service, the request for a certificate fails.  
+
+challengeRetries=Challenge retries
+challengeRetries.desc=The number of times to retry updating the challenge status before aborting the challenge.
+
+challengeRetryWait=Challenge retry wait
+challengeRetryWait.desc=The amount of time to wait before retrying to refresh the status of a challenge.
+
+orderRetries=Order retries
+orderRetries.desc=The number of times to retry updating the order status before aborting the order.
+
+orderRetryWait=Order retry wait
+orderRetryWait.desc=The amount of time to wait before retrying to refresh the status of an order.
+
+accountContact=Account contact
+accountContact.desc=A contact URL the ACME server can use to contact the client for issues related this the ACME account.
+
+accountKeyFile=Account key file
+accountKeyFile.desc=A path to the file containing a key identifier for a registered account on the ACME CA server. If the file does not exist, a new account is registered with the ACME CA server and the associated key is written to this file. Back this file up to maintain control of the account on the ACME CA server. 
+
+domainKeyFile=Domain key file
+domainKeyFile.desc=A path to the file containing a key identifier for a domain. If the file does not exist, a new key will be generated and written to this file. Back this file up to maintain control of the domain. 
```

Comment:
**change**
If the file does not exist, a new key will be generated and written to this file.
**to**
If the file does not exist, a new key is generated and written to this file.

Reply:
Made the change.